### DISCUSSION TITLE
"Filter by repository" on Inbox page

### DISCUSSION THREAD

**Discussion by amadejpapez** [community/community#17517] [Mobile]
Option to Filter by Repository on the Inbox main page seems to create a  repository list from the ones I received notifications from.

I noticed that if I subscribe to an Issue/PR, receive notifications for some time,.. But once the issue is solved, I unsubscribe, the notifications will disappear but the repo will remain in the filter list.

So I see quite a long filter list with most showing zero notifications when selected.

Am I understanding this feature correctly and is it intentional to work like this?

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.67.0 (68716861)
Device: iPhone 8 Plus (iOS 15.5)
</details>

**Comment by HenSquared** [comment]
Hi @amadejpapez,

Thank you for your feedback. I'm able to reproduce this issue as well. 
I've opened up an issue to discuss and work on this internally.

Thanks!
